Scripts/update_ncbi_codon_table.py

```diff
@@ -32,13 +41,18 @@ def line_wrap(text, indent=0, max_len=78):
         return line + "\n" + line_wrap(rest, max_len)
 
 
-print("##########################################################################")
-print("# Start of auto-generated output from Scripts/update_ncbi_codon_table.py #")
-print("##########################################################################")
-print("")
-print("")
+print("""
+##########################################################################
+# Start of auto-generated output from Scripts/update_ncbi_codon_table.py #
+##########################################################################
+
+"""[1:])
```

Comment:
Personally I found the five separate print statements clearer, but don't have a strong view on this change.